scripts/actions/notifications.js

```diff
@@ -28,9 +28,7 @@ export function notifySuccess(message, options) {
 }
 
 export function notifyError(error, options) {
-  if (process.env.NODE_ENV !== "production") {
-    console.error(error);
-  }
+  console.error(error);
```

Comment:
Yes, I realized that reporting bugs is a pain if users can't simply inspect the console and tell us what went wrong precisely.